pzsd_bot/__init__.py

```diff
@@ -0,0 +1,17 @@
+import logging
+from logging import handlers
+from pathlib import Path
+
+from pzsd_bot.constants import Bot
+
+logger = logging.getLogger()
+logger.setLevel(Bot.log_level)
+log_file = Path(__file__).parents[1] / Path("logs", "pzsd_bot.log")
+log_file.parent.mkdir(exist_ok=True)
+handler = handlers.RotatingFileHandler(
+    log_file, maxBytes=1024 * 1024 * 3, backupCount=2, encoding="utf8"
```

Comment:
prety sure utf 8 is the default


3 * 1024**2 is cuter

Reply:
Looks to be `None` actually: https://docs.python.org/3/library/logging.handlers.html#logging.handlers.RotatingFileHandler

You're right on the second point though

Reply:
If it's None does it use system default? Has to choose some encoding

Reply:
I had the same thought, but was too lazy to look any further into it. Doesn't hurt to be explicit.